tools/tplg_parser/include/tplg_parser/topology.h

```diff
@@ -53,6 +53,7 @@ enum sof_ipc_process_type {
 	SOF_PROCESS_MUX,
 	SOF_PROCESS_DEMUX,
 	SOF_PROCESS_DCBLOCK,
+	SOF_PROCESS_CROSSOVER,
```

Comment:
looks like you accidentally squashed in a few of seb's changes, this will need to be removed